### ISSUE TITLE
Rename Page titles on Progress and Rankings pages

### ISSUE THREAD

**Issue by rmmayo** [NationalSecurityAgency/skills-service#1508]
The page title for the `/progress-and-rankings/` page should be "Progress and Rankings" and not "My Progress"

The page title for the `/progress-and-rankings/manage-my-projects/` page should be should be "Manage My Projects" and not "Discover Projects"

**Comment by rmmayo** [comment]
PR: #1510

**Comment by sudo-may** [comment]
:+1:
